app/components/TaskTemplateListColumn.tsx

```diff
@@ -0,0 +1,66 @@
+import { tw } from '@helpwave/common/twind/index'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import { useState } from 'react'
+import { TaskTemplateTile } from './TaskTemplateTile'
+import Add from '../icons/Add'
+
+export type TaskTemplateListColumnTranslation = {
+  name: string,
+  addNewTaskTemplate: string
+}
+
+type TaskTemplateDTO = {
+  name: string,
+  subtaskCount: number
+}
+
+export type TaskTemplateListColumnProps = {
+  translations: Record<Languages, TaskTemplateListColumnTranslation>,
```

Comment:
Why do you propagate the translations back to the caller? Is there any reason why not handle the translations the same as in https://github.com/helpwave/web/blob/issue/116-Stacked_Profile_Pictures_Component/app/components/StackedProfiles.tsx for example?

Reply:
Because the translation is based upon the name of the column for which there are at least 2 different cases. Allowing the caller to set the translation allows an easy addition of another column.

But I will make a case distinction for both possible column names, since the likelyhood of an additional one in the near future is low and not worth violating the pattern